README.md

```diff
@@ -170,6 +170,12 @@ This list results from Pull Requests, reviews, ideas, and work done by 1600+ peo
     * [Pages](https://developers.cloudflare.com/pages/) - Develop and deploy your web apps on Cloudflare's fast, secure global network. Five hundred monthly builds, 100 custom domains, Integrated SSL, unlimited accessible seats, unlimited preview deployments, and full-stack capability via Cloudflare Workers integration.
     * [Queues](https://developers.cloudflare.com/queues/) - 1 million operations per month
     * [TURN](https://developers.cloudflare.com/calls/turn/) – 1TB of free (outgoing) traffic per month.
+  
+  * [Gcore](https://gcore.com/)
```

Comment:
I am not convinced this qualifies as a "major cloud provide", not even linode, digital ocean etc listed here and those are much more significant than core, so I prefer we just collapse this to a single line entry where it was a mention the other things thats free as a short extra bit there

Reply:
Cloudflare is used by 80% of the worlds websites, has a large free tier offering many services on a truly global scale.

What would be your definition of major provider if not 80% of the world using them?

Reply:
There is no way you'd compare Cloudflare and this convincingly.

Please flatten the entry or close the PR.

Reply:
It's not a ethics contest either.

As per the code of conduct, this isn't a place to argue over the merits of listings, I'll close and reject the PR accordingly thank you.

Reply:
```### Code of Conduct

 We are not here to argue with you. If you are argumentative, abusive,
 lie or missrepresent your service or are otherwise anti-social we will
 block you.```